lib/Runtime/Library/BuiltIn/IndexOf.js

```diff
@@ -0,0 +1,48 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+"use strict";
+
+(function indexOf(searchElement, fromIndex) {
+
+    var k;
+
+    if (this == null) {
+        throw new TypeError('"this" is null or not defined');
+    }
+
+    var o = this;
+
+    var len = o.length >>> 0;
```

Comment:
> Edit: actually, that's not true anymore. It used to be that we need to call ToUint32, now we need to do ToLength, so our coercing ToUint32 isn't valid.

It seems the difference is that any negative value for lenValue must become 0, but since an array cannot produce a lenValue < 0 it doesn't matter. Additionally, the higher upper range for length is not necessary to consider because Array length is capped by the spec at 2^32-1 anyway.

IOW, <<<0 will do the right thing even though the semantics in the spec have changed.